src/d/actor/d_a_mozo.cpp

```diff
@@ -112,192 +222,159 @@ void daMozo_c::setAnm(int, float) {
 }
 
 /* 00001F70-00002228       .text CreateInit__8daMozo_cFv */
-void daMozo_c::CreateInit() {
+s32 daMozo_c::CreateInit() {
     /* Nonmatching */
-}
+    J3DModelData* mdlData = mAnimMorf->getModel()->getModelData();
 
-/* 00002228-000023B0       .text _create__8daMozo_cFv */
-void daMozo_c::_create() {
-    /* Nonmatching */
-}
+    int param = mBase.mParameters * 0xFF;
+    if (param == 0xFF) {
+        param = 0;
+    }
+    
+    int a = 2;
+    if (param < 3) {
+        a = param;
+    }
 
-/* 000023B0-0000242C       .text _delete__8daMozo_cFv */
-void daMozo_c::_delete() {
-    /* Nonmatching */
-}
+    //m0376 = a;
 
-/* 0000242C-00002498       .text getBeamActor__8daMozo_cFUi */
-void daMozo_c::getBeamActor(unsigned int) {
-    /* Nonmatching */
-}
-
-/* 00002498-00002588       .text event_move__8daMozo_cFv */
-void daMozo_c::event_move() {
-    /* Nonmatching */
-}
-
-/* 00002588-000025DC       .text _execute__8daMozo_cFv */
-void daMozo_c::_execute() {
-    /* Nonmatching */
-}
-
-/* 000025DC-0000267C       .text _draw__8daMozo_cFv */
-void daMozo_c::_draw() {
-    /* Nonmatching */
-}
-
-/* 0000267C-000026A0       .text daMozo_Draw__FP8daMozo_c */
-void daMozo_Draw(daMozo_c*) {
-    /* Nonmatching */
-}
-
-/* 000026A0-000026C4       .text daMozo_Execute__FP8daMozo_c */
-void daMozo_Execute(daMozo_c*) {
-    /* Nonmatching */
-}
-
-/* 000026C4-000026CC       .text daMozo_IsDelete__FP8daMozo_c */
-void daMozo_IsDelete(daMozo_c*) {
-    /* Nonmatching */
-}
-
-/* 000026CC-000026F0       .text daMozo_Delete__FP8daMozo_c */
-void daMozo_Delete(daMozo_c*) {
-    /* Nonmatching */
-}
-
-/* 000026F0-00002710       .text daMozo_Create__FP10fopAc_ac_c */
-void daMozo_Create(fopAc_ac_c*) {
-    /* Nonmatching */
-}
+    mAnimMorf->getModel()->setUserArea(this);
+    for (int i = 0; i < mdlData->getJointNum(); i++) {
+        if (i == 2) {
+            mdlData->getJointNodePointer(2)->setCallBack((J3DNodeCallBack)daMozo_nodeCallBack);
+        }
+    }
 
-/* 00002710-000027AC       .text __dt__12daMozo_HIO_cFv */
-daMozo_HIO_c::~daMozo_HIO_c() {
-    /* Nonmatching */
-}
+    //mQuatRotation = ZeroQuat;
```

Comment:
needed?

Reply:
Forgot about the mozo work I did, I will clean it up real quick.

Reply:
Added some context to those comments.